apps/backend/api/graphql/mutations/user.js

```diff
@@ -45,31 +45,31 @@ export const verifyEmail = (fetchOne, { req }) => async (_, { email: providedEma
 
     await user.save({ email_validated: true })
 
-    req.session.userId = user.id
+    context.req.session.userId = user.id
```

Comment:
if we are switching to context.currentUserId everywhere then why do we set context.req.session.userId here?

Reply:
It's a good question. We are continuing to set the actual userId of the session on the actual Express request object. The currentUserId that is in context is a copy of that value (see the createYoga#context function in graphql/index). Setting it won't reset the session to this token auth's user, but setting the req.session.userId does (evidently). We could also reset the currentUserId on context here, but it will already be reset to this value upon the next request to the server by this user.